CHANGELOG.md

```
@@ -1,192 +1,60 @@
-# Run tests — all 128 should pass
-npx vitest run
-
-# Verify no console.log in source
-grep -r "console.log" src/ --include="*.ts"
-# Should return empty
-```
+# Changelog
+
+## [Unreleased] - 2026-05-28
+
+### Added
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)